src/Plot.js

```diff
@@ -184,6 +184,7 @@ export default class Plot extends Viz {
         }, {});
 
       data = data.sort((a, b) => {
+        if (this[`_${this._discrete}Sort`]) return this[`_${this._discrete}Sort`](a, b);
```

Comment:
When calling the user's sort method, make sure to pass them their original data points: `(a.data, b.data)`